model/example_domains_test.go

```diff
@@ -0,0 +1,492 @@
+package model_test
+
+import (
+	"fmt"
+
+	"github.com/nextmv-io/sdk/model"
+)
+
+func ExampleNewDomains_first() {
```

Comment:
The same comment here regarding documenting the methods of the interface and what they do by having `ExampleDomains_add`, `ExampleDomains_assign`, etc... This would mirror what is currently on  `store` to have some consistency.